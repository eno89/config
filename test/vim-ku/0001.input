" Test to test the test system.

echo 'foo'
ls
cho 'bar'
echo 'baz'
echo ''

" __END__
