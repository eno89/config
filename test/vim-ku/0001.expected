foo
  1 %a   "[No Name]"                    line 1
Error detected while processing /test/vim-ku/0001.input:
line    5:
E492: Not an editor command: cho 'bar'
baz
